packages/import/src/helpers/labelManager.ts

```diff
@@ -196,7 +196,8 @@ class LabelManager {
    * @returns Label instance if found
    */
   public getLabelByName(name: string, teamId: Id | typeof WORKSPACE_ID): Label | undefined {
-    return this.nameToLabel[name.toLowerCase()]?.[teamId] ?? this.nameToLabel[name]?.[WORKSPACE_ID];
+    const normalized = Label.normalizeName(name);
+    return this.nameToLabel[normalized]?.[teamId] ?? this.nameToLabel[normalized]?.[WORKSPACE_ID];
```

Comment:
When determining whether a particular label already exists, `getLabelByName` will now normalize the name so that `foo` and ` foo` are consolidated.